tools/nl/embeddings/.gitignore

```diff
@@ -0,0 +1 @@
+!input/
```

Comment:
Can we move this to the root .gitignore so there is one place to track?

Reply:
The root ignores it so there may be input folders that may be ignored elsewhere.

Reply:
Looks like there is a rule from https://github.com/datacommonsorg/website/pull/3472/files, maybe make that more specific?

Reply:
Thanks - done. Looks like the specific rule is [already there](https://github.com/datacommonsorg/website/blob/master/server/integration_tests/standalone/.gitignore) so removed the root one. With this we don't need a `.gitignore` in this folder anymore.

Reply:
Thanks - done. Looks like the specific rule is [already there](https://github.com/datacommonsorg/website/blob/master/server/integration_tests/standalone/.gitignore) so removed the root one. With this we don't need a `.gitignore` in this folder anymore.